lib/validators.ts

```diff
@@ -101,9 +101,17 @@ export const generateFlexibleSchema = (
       );
     } else if (field.type === 'timetable') {
       shape[field.id] = Yup.object();
+    } else if (field.type === 'date' && field.isfutureDateValid === false) {
+      shape[field.id] = Yup.string().test(
+        'Validate date is present or past',
+        'Date cannot be in the future',
```

Comment:
So I understand the second string but what's the first string for?

Reply:
The first string it's the test name/description 😄